src/components/organization/OrganizationsPartnerList.js

```diff
@@ -0,0 +1,154 @@
+import {Controller, useForm} from "react-hook-form";
+import {Box, Button, FormControl, Grid, IconButton, InputLabel, MenuItem, Select, TextField} from "@material-ui/core";
+import {withSwal} from "../formsCommons/WithSwal";
+import React, {useEffect} from "react";
+import ClearIcon from "@material-ui/icons/Clear";
+import {OrganizationPartner} from "./OrganizationPartner";
+import {usersAPI} from "../../utils/apis/UserApi";
+
+export const OrganizationsPartnerList = (props) => {
+
+    const id = 1 //TODO change after jwt
+    function updateUsers(data) {
+        data.map((el => {
+            users.push({firstName: el.user.firstName, lastName: el.user.lastName, mail: el.user.mail})
+        }))
```

Comment:
Nie podoba mi się to pushowanie, bo chyba ta funkcja nie będzie używana do dopisywania userów w trakcie, tylko do pobrania z responsa, poza tym wystarczy zrobić chyba el.user
```suggestion
        setUsers(data.map(el => el.user))
```
(To przy założeniu, że users są z useState)